integration/network/network.go

```diff
@@ -4,61 +4,18 @@ import (
 	"fmt"
 	"net"
 
-	"github.com/giantswarm/ipam"
-	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/azure-operator/service/controller/v3/network"
 )
 
 const (
 	e2eNetwork        = "11.%d.0.0"
 	e2eSubnetQuantity = 256
-
-	azureCalicoSubnetMask = 17
-	azureMasterSubnetMask = 24
-	azureWorkerSubnetMask = 24
 )
 
-type azureCIDR struct {
-	AzureCIDR        string
-	MasterSubnetCIDR string
-	WorkerSubnetCIDR string
-	CalicoSubnetCIDR string
-}
-
-func ComputeCIDR(buildNumber uint) (*azureCIDR, error) {
-	cidrs := new(azureCIDR)
-
+func ComputeSubnets(buildNumber uint) (*network.Subnets, error) {
 	azureNetwork := determineSubnet(e2eNetwork, e2eSubnetQuantity, buildNumber)
```

Comment:
I'd say we don't need to extract `determineSubnet` anymore :)

Reply:
We still need this to get e2e test network